features/netsocket/emac-drivers/TARGET_Freescale_EMAC/kinetis_emac.cpp

```diff
@@ -504,19 +577,20 @@ bool Kinetis_EMAC::power_up()
     rx_isr();
 
     /* PHY monitoring task */
-    prev_state.connected = STATE_UNKNOWN;
+    prev_state.connected = STATE_LINK_DOWN;
     prev_state.speed = (phy_speed_t)STATE_UNKNOWN;
     prev_state.duplex = (phy_duplex_t)STATE_UNKNOWN;
 
-    phy_task_handle = mbed::mbed_event_queue()->call_every(PHY_TASK_PERIOD_MS, mbed::callback(this, &Kinetis_EMAC::phy_task));
+    mbed::mbed_event_queue()->call(mbed::callback(this, &Kinetis_EMAC::phy_task));
```

Comment:
Why is this added?

Reply:
The first call_every() callback is triggered after 200ms. Now the first call() callback is triggered right after the power up and then every 200ms.